src/config/display.c

```diff
@@ -342,5 +274,24 @@ u8 CONFIG_ReadDisplay()
     #else
     char filename[] = "media/config.ini";
     #endif
-    return CONFIG_IniParse(filename, ini_handler, (void *)&Display);
+    if (CONFIG_IniParse(filename, ini_handler, (void *)&Display)) {
+        for (int i = 0; i < NUM_LABELS; i++) {
+            struct LabelDesc *label = &Display.font[i];
+            // For compatibility reasons,
+            // use the old alignment values as default
+            // if the new one is not yet set:
+            if (label->align == 0) {
+                switch (label->style) {
+                    case LABEL_CENTER: label->align = ALIGN_CENTER; break;
+                    case LABEL_LEFT:   label->align = ALIGN_LEFT;   break;
+                    case LABEL_RIGHT:  label->align = ALIGN_RIGHT;  break;
+                    default: break;
+                }
+            }
+        }
+
+        return 1;
+    }
+
```

Comment:
This is ugly

Reply:
this is not new code. I just moved the code around. I think this sort of ugly is caused by back-compatibility. I am not sure when we can remove it.

Reply:
I'm not concerned with the code so much as where it is placed.  However, we can find a good place for it later once you have everything working